code/src/reducers/game.js

```diff
@@ -0,0 +1,64 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { ui } from './ui';
+
+export const game = createSlice({
+  name: 'game',
+  initialState: {
+    username: '',
+    gameList: {
+      coordinates: '',
+      description: '',
+      actions: []
```

Comment:
Do you need to specify this if it's all going to be overwritten by setGameElements?